src/components/ec-btn-dropdown/ec-btn-dropdown.vue

```diff
@@ -28,6 +30,23 @@
       @open="isOpen = true"
       @close="isOpen = false"
     >
+      <template #item="{ item }">
```

Comment:
We need to test this template, mount a component with items that have one item disabled, one with `href`, one with `to` and one with `attrs`. We can pass an random ID in the `id` and a clickSpy in `click`, check the ID is rendered on the item, check that spy is called when item is clicked.

Reply:
corrected